packages/core/package.json

```diff
@@ -6,7 +6,9 @@
     "@code-pushup/models": "*",
     "@code-pushup/utils": "*",
     "@code-pushup/portal-client": "^0.6.1",
-    "chalk": "^5.3.0"
+    "chalk": "^5.3.0",
+    "memfs": "^4.5.0",
+    "vitest": "1.3.1"
```

Comment:
:no_entry: Neither `memfs` or `vitest` are prod dependencies. Are you getting some Nx lint error?

Reply:
OOhhh good catch! that should throw a lint error if unused... hmm..